README.md

```
@@ -1,7 +1,151 @@
+```bash
+git tag v0.11.0
+git push origin v0.11.0
+```
+
+The `Publish` job in CI then runs cibuildwheel, uploads the wheels + sdist
+to PyPI via Trusted Publishing (OIDC, no tokens), and creates a GitHub Release
+with the artifacts attached.
+
+## Layout
+
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)